README.md

```diff
@@ -2,10 +2,10 @@
 
 The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to enable the creation of interoperable tools for building, transporting, and preparing a container image to run.
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](serialization.md).
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
 At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem.
```

Comment:
Also later in this file, “[…the critical filesystem serialization and image manifest described…][1]” which should probably be updated to include the “image manifest, filesystem layer, and configuration” wording we're using around here.

[1]: https://github.com/opencontainers/image-spec/blame/7e6e2f76d6a11cdcb3f3e334e3f12179a0f37dad/README.md#L20